org.dharkael.Flameshot.yaml

```diff
@@ -0,0 +1,36 @@
+app-id: org.dharkael.Flameshot
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+command: flameshot
+rename-icon: flameshot
+rename-appdata-file: flameshot.appdata.xml
```

Comment:
These renames should also happen in upstream repository.

Reply:
removed